### ISSUE TITLE
.eslintIgnore file seems invaild!

### ISSUE THREAD

**Issue by stevennzhou** [eslint/eslint#10766]
**Tell us about your environment**

* **ESLint Version:** 4.19.1
* **Node Version:** v8.9.3
* **npm Version:** 6.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  extends: ['eslint-config-alloy/typescript'],
  globals: {},
  rules: {
    indent: ['error', 2],
    semi: ['error', 'never'],
    'no-param-reassign': 0,
    'typescript/member-delimiter-style': 'none'
  }
}
```
</details>

**Something Strange**

I use .eslintignore file to ignore folder, but it is invalid.The working tree just like this:
![image](https://user-images.githubusercontent.com/19498847/44185272-def1bd00-a145-11e8-9311-97c645cdaff6.png)

.eslintignore config is:
```js
dist/
```

And then I open the express-dev in VSCode, .eslintignore is vaild!
![image](https://user-images.githubusercontent.com/19498847/44185422-a7cfdb80-a146-11e8-801c-69585adc3d87.png)

It seems a bug!

**Comment by kaicataldo** [comment]
It's not clear to me what the issue you're experiencing is. Mind explaining what the expected and actual behaviors are here? Thanks!
